scipy/stats/tests/test_stats.py

```diff
@@ -4413,7 +4413,7 @@ def test_chisquare_12282a(self, xp):
         with assert_raises(ValueError, match='For each axis slice...'):
             f_obs = xp.asarray([10., 20.])
             f_exp = xp.asarray([30., 60.])
-            stats.chisquare(f_obs=f_obs, f_exp=f_exp)
+            stats.chisquare(f_obs, f_exp=f_exp)
```

Comment:
I missed this in the first CI run, and it was the cause of the only failure, so I went ahead and skipped CI on this commit.